examples/relayWebhooks/delete_relayWebhook.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var key = 'YOURAPIKEY'
+  , SparkPost = require('sparkpost')
+  , client = new SparkPost(key)
+  , webhookId = '123456789';
```

Comment:
let's call this `relayWebhookId` so that we don't cause confusion with regular webhooks :) (same for the other example files)